nss-day-cohort-51/nss-day-cohort-51.github.io
# mw-bug-coming-soon-image-capstone-vid copy and pasted the image alien…

# fixed broken link
copy and pasted the image alien-coming-soon-340w-235h.gif into the public folder; 
now, the coming soon image shows again on student Learn More modals where the student doesn't have a capstone video

## Type of change


# Testing Instructions

1. pull down branch and run npm run
2. click on a student card without a capstone video -- for example, Michael Wright. Click Learn More to open the modal
3. See that the alien head coming soon image is under the capstone title  \

# Checklist:
